### DISCUSSION TITLE
elsa aspi route prefixes

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5445] [Q&A]
How to modify elsa aspi route prefixes

**Comment by glime-ai** [comment]
To modify the route prefixes for Elsa API endpoints, you can change the default prefix path by passing a different prefix path to the `UseHttp` method in your ASP.NET Core application's startup configuration. Here's an example of how to change the prefix to `/api`:

```csharp
elsa.UseHttp(http => http.ConfigureHttpOptions = options =>
    options.BasePath = "/api");
```

This code snippet should be placed in the `ConfigureServices` method within your `Startup` class or the `Program.cs` file, depending on the version of ASP.NET Core you are using.

Please ensure that you have the necessary using directives at the top of your file to reference the Elsa namespaces required for this configuration.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
